playbooks/deploy_http_int.yml

```diff
@@ -4,9 +4,7 @@
   - role: scp_http_int
   - role: httpd
     website_domain: http.int.rht.gluster.org
-    document_root: "/home/logs/glusterfs-logs"
+    document_root: /var/www/glusterfs-logs
     aliases:
```

Comment:
So, why do we want a alias if we have the document_root pointing directly here ?

Reply:
Yeah, we don't really need alias in this case.I'll update it.